### PR TITLE
scheduler: enable `evict-slow-store` by default.

### LINKED ISSUE #7564: scheduler: enable `evict-slow-store` scheduler by default.
## Development Task

In previous works mentioned in https://github.com/tikv/tikv/issues/15909, we've:
- Fine-tune the Slow-Score detection algorithm to reduce the false-positive rate of Slow-Score detection algorithm.
- Reduce the reaction interval by modifying the default sampling interval of Slow-Score from 500ms to 100ms.

And, by testing, the fine-tuned SlowScore detection algorithm has considerable sensitiveness to Disk IO jitters, and lower false-positive rate to hotspots cases. So, it's the time to enable it by default.

### PR DESCRIPTION
**Pull request by LykxSassinator**
### What problem does this PR solve?

Issue Number: close #7564, ref https://github.com/tikv/tikv/issues/15909

### What is changed and how does it work?

```commit-message
Enable `evict-slow-store` scheduler by default.
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Code changes

- Has the configuration change
- Has HTTP API interfaces changed (Don't forget to [add the declarative for the new API](https://github.com/tikv/pd/blob/master/docs/development.md#updating-api-documentation))
- Has persistent data change

Side effects

- Possible performance regression
- Increased code complexity
- Breaking backward compatibility

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tiup`](https://github.com/pingcap/tiup):
- Need to cherry-pick to the release branch

### Release note

```release-note
None.
```

### CODE REVIEW

File: server/cluster/cluster_test.go [hunk 1]
```diff
@@ -3185,9 +3188,9 @@ func TestPersistScheduler(t *testing.T) {
 
 	co.Run()
 	controller = co.GetSchedulersController()
-	re.Len(controller.GetSchedulerNames(), defaultCount-1)
+	re.Len(controller.GetSchedulerNames(), 4)
```
[the remark this answers is not in the document]
Comment (LykxSassinator) [reply server/cluster/cluster_test.go:3191]: IMO, just using `defaultCount` is not appropriate in this case if we add more default schedulers.

So, I replaced all related reference to `defaultCount` with the precise value.

**Review by rleungx — APPROVED** [review]

**Review by bufferflies — APPROVED** [review]

**Pull request merged**